src/components/ApplicationsModal/ApplicationsModal.tsx

```diff
@@ -258,18 +263,22 @@ const ApplicationsModal: React.FC<ApplicationsModalProps> = ({
 
     if (indexOfFormFieldToRemove === 0 && formState.values.appTermsURL) {
       formState.values.appTermsURL = ''
+      // @ts-ignore
```

Comment:
this a pretty much common action, if the rules don’t allow you to do it, update the rules instead. Try to avoid `@ts-ignore` as much as possible.

Or in the case `formState.errors` is possibly null do `formState?.errors?.appPrivacyURL`